source/_static/css/custom.css

```diff
@@ -17,3 +17,10 @@
 .wy-nav-top {
   background: #C8102E;
 }
+
+span.gp {
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}
```

Comment:
It might be good to namespace this so that it doesn't potentially affect anything else in the future.

If every code block highlighted with console is wrapped in a div with the class "highlight-console" maybe this is appropriate:

```css
div.highlight-console span.gp {
}
```